app/assets/stylesheets/application.css.sass

```diff
@@ -33,7 +36,8 @@ body
     right: 0
     top: 0
     height: 66px
-    background-color: rgb(222, 57, 12)
+    //background-color: rgb(222, 57, 12)
```

Comment:
Can you remove this comment? If we need to revert to it, we have it in Git :smile: